### PR TITLE
Dalle

### PR DESCRIPTION
**Pull request by Mole1424**
a lot cleaner code base (condensed files)
Now also starts a new message chain after 10 messages sent

### CODE REVIEW

**Comment by Mole1424** [comment]
also tempted to add a thing which disallows onion from using it

File: cogs/commands/dalle.py [hunk 1]
```diff
@@ -58,10 +58,9 @@ async def dalle(self, ctx: Context, *, prompt: str):
                 "Failed to generate image :wah:", mention_author=True
             )
         view = DalleView(timeout=None, bot=self.bot)  # otherwise rpley with image
```
Comment (ericthelemur) [inline cogs/commands/dalle.py:60]: Typo in comment

File: cogs/commands/dalle.py [hunk 2]
```diff
@@ -83,75 +82,67 @@ async def generate_variant(image):
...
+    async def variant(
+        self, interaction: discord.Interaction, button: discord.ui.Button
+    ):
         """generates a variant of the image"""
-        logging.info("generating variant")
-        self.edit_buttons(True)
-        message = interaction.message
+        await self.new_image(interaction, "Creating variant")
+
+    async def new_image(self, interaction: discord.Interaction, mode: str):
```
Comment (ericthelemur) [inline cogs/commands/dalle.py:103]: I'd prefer `mode` as an enum instead of str

**Review by ericthelemur — COMMENTED** [review]
Only one actual change, looking good otherwise

**Comment by ericthelemur** [comment]
tested and looks good, will merge and deploy

**Pull request merged**